README.md

```diff
@@ -212,6 +212,11 @@ Forest developers will prepend this variable to CLI commands over using `export`
 FULLNODE_API_INFO="..." forest auth api-info -p admin
```

Comment:
It'd be great to update it to the new CLI binary, i.e. `FULLNODE_API_INFO="..." forest-cli auth api-info -p admin`

Reply:
corrected